src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -188,13 +190,13 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
           remove={removeDeliverable}
         />
       </PageBlock>
-      <Box textAlign="center" sx={{ my: 2 }}>
-        <Button variant="contained" color="success" type="submit" sx={{ mx: 2 }}>
-          Submit
-        </Button>
-        <Button variant="contained" color="error" onClick={exitEditMode} sx={{ mx: 2 }}>
+      <Box textAlign="right" sx={{ my: 2 }}>
+        <NERFailButton variant="contained" onClick={exitEditMode} sx={{ mx: 2 }}>
           Cancel
-        </Button>
+        </NERFailButton>
+        <NERSuccessButton variant="contained" type="submit" sx={{ mx: 2 }}>
```

Comment:
mx should be 1 on both of these to be standard with how we're doing the rest of these buttons